features/step_definitions/my_google_steps.rb

```diff
@@ -1,82 +1,46 @@
 end
 
 Then(/^I see the Google Search Results Page$/) do
-  pending # express the regexp above with the code you wish you had
-  #Hint: how do you know you're on a different page than before?
-  #https://github.com/jnicklas/capybara#navigating
-
-  #Then steps are most often going to involve RSpec expectation matchers
-  #Which take this form
-  #expect(current_url).to contain(expected_string)
-  #expect(current_url).to match(/\?some_param=\d+/)
-  #https://relishapp.com/rspec/rspec-expectations/docs
-
-  #SitePrism Hint: Get an instance of GoogleSearchResults. If your GoogleSearchResults class has set_url_matcher, how can you assert that the page url is loaded?
-  #SitePrism Hint: If your GoogleSearchResults class has elements defined, how can you use one assert to ensure all defined elements are present?
+  $app.search_results.load(query: 'Kittens')
+  $app.search_results.displayed?
 end
 
-Then(/^the Google Search Results Page has Kittens$/) do
-  pending # express the regexp above with the code you wish you had
-  #Hint: how can you find out if the word Kitten occurs on the page somewhere?
-  #https://github.com/jnicklas/capybara#querying
+Then(/^the (.*) Page has Kittens$/) do |page_name|
+  expect($app.send(page_name).text).to have_content('Kittens')
+end
 
-  #SitePrism Hint: What element(s) could you define on the GoogleResultsPage object that would always have text reflecting your search term?
-  #SitePrism Hint: Especially if you're checking multiple elements for your search term, what method could you write to perform a singular check for accurate search results?
+When(/^I click the first result on the Google Search Results page$/) do
+  $app.search_results.click_first_search_result
 end
 
+Then(/^I am redirected to the corresponding website from the search result$/) do
+  expect(current_url).to include($app.search_results.first_search_result_url)
+end
+#
 When(/^I click the News Tab on the Google Search Results Page$/) do
-  pending # express the regexp above with the code you wish you had
-  #Hint: you should have the hand of finding and clicking elements by now
-
-  #SitePrism Hint: Make sure you've defined the various tabs, such as News, as an element on GoogleSearchResults
+  $app.search_results.click_news_tab
 end
 
 Then(/^I see the Google News Results Page$/) do
-  pending # express the regexp above with the code you wish you had
-  #Hint: you should have the hang of finding out where you are according to url by now
-
-  #SitePrism Hint: You can assert the page is loaded and all expected elements exist just as you did above with Google Search Results
-end
-
-Then(/^the Google News Results Page has Kittens$/) do
-  pending # express the regexp above with the code you wish you had
-  #Hint: you should have the hang of querying a page by now
-
-  #SitePrism Hint: You can assert this page reflects your search term just as you did above with Google Search Results
+  $app.news_results.load(query: 'Kittens')
```

Comment:
If this step comes after you click the news tab, should you have to actually load the news results page? This step should pass with just asserting:
```ruby
expect($app.news_results).to be_displayed
#or
expect($app.news_results.displayed?).to be_truthy
```